src/Agent.Worker/CodeCoverage/CodeCoverageCommands.cs

```diff
@@ -270,6 +272,27 @@ private void LoadPublishCodeCoverageInputs(Dictionary<string, string> eventPrope
             }
         }
 
+        /// <summary>
+        /// This method replaces the default index.html generated by cobertura with 
+        /// the non-framed version
+        /// </summary>
+        /// <param name="reportDirectory"></param>
+        private void ModifyCoberturaIndexDotHTML(string reportDirectory, IExecutionContext executionContext)
+        {
+            try
+            {
+                if (_codeCoverageTool.Equals("cobertura", StringComparison.OrdinalIgnoreCase) && File.Exists(Path.Combine(reportDirectory, CodeCoverageConstants.DefaultIndexFile)) && File.Exists(Path.Combine(reportDirectory, CodeCoverageConstants.DefaultNonFrameFileCobertura)))
+                {
+                    File.Move(Path.Combine(reportDirectory, CodeCoverageConstants.DefaultIndexFile), Path.Combine(reportDirectory, CodeCoverageConstants.NewIndexFile));
+                    File.Move(Path.Combine(reportDirectory, CodeCoverageConstants.DefaultNonFrameFileCobertura), Path.Combine(reportDirectory, CodeCoverageConstants.DefaultIndexFile));
```

Comment:
You have not considered changes in [this](https://mseng.visualstudio.com/VSOnline/Test%20Infrastructure/_git/VSO/pullrequest/110815#view=discussion&fullScreen=false) and [this](https://mseng.visualstudio.com/VSOnline/Test%20Infrastructure/_git/VSO/pullrequest/112413?view=discussion) PR. Please incorporate these changes.